src/hotspot/os/linux/os_linux.cpp

```diff
@@ -4921,14 +4921,18 @@ int os::create_binary_file(const char* path, bool rewrite_existing) {
   return ::open64(path, oflags, S_IREAD | S_IWRITE);
 }
 
+off64_t call_lseek64(int fd, off64_t offset, int whence) {
```

Comment:
I think it would be better to just change the `lseek64` calls to `os::lseek` rather than introduce this wrapper function.

Reply:
Thanks David.  I changed the code to call os::lseek().